src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLScalarFunctionsIT.java

```diff
@@ -0,0 +1,39 @@
+package com.exasol.adapter.dialects.postgresql;
+
+import java.sql.*;
+import java.util.Map;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import com.exasol.adapter.dialects.ScalarFunctionsAbstractIT;
+import com.exasol.closeafterall.CloseAfterAll;
+import com.exasol.closeafterall.CloseAfterAllExtension;
+import com.exasol.dbbuilder.dialects.Schema;
+import com.exasol.dbbuilder.dialects.exasol.VirtualSchema;
+
+@ExtendWith({ CloseAfterAllExtension.class })
+class PostgreSQLScalarFunctionsIT extends ScalarFunctionsAbstractIT {
+    private static final String EMPTY_POSTGRES_TABLE = "empty_postgres_table";
+    @CloseAfterAll
+    private final PostgresVirtualSchemaIntegrationTestSetup setup = new PostgresVirtualSchemaIntegrationTestSetup();
+
+    // TODO capital letters cause crash; bug?
```

Comment:
This is interesting. How can we reproduce it?

Reply:
just change column name for example to floatingPoint.

Reply:
I removed that comment and created an issue: https://github.com/exasol/postgresql-virtual-schema/issues/21